app/assets/javascripts/directives/views/accountMenu.js

```diff
@@ -377,6 +377,17 @@ class AccountMenuCtrl extends PureCtrl {
   }
 
   async performImport(data, password) {
+    if (data.keyParams) {
+      if (Number(data.keyParams.version) > 3) {
+        this.setState({
+          importData: null
+        });
+        this.alertManager.alert({
+          text: "This backup file was created using a later version of the application and cannot be imported."
```

Comment:
"This backup file was created using a newer version of the application and cannot be imported here. Please update your application and try again."